src/@gen3/arranger/lib/config.ts

```diff
@@ -46,9 +48,38 @@ if (process.env['GEN3_ARBORIST_ENDPOINT']) {
 if (process.env['GEN3_AUTH_FILTER_FIELD']) {
   singleton.authFilterField = process.env['GEN3_AUTH_FILTER_FIELD'];
 }
-if (process.env['GEN3_AUTH_FILTER_NODE_TYPE']) {
-  singleton.authFilterNodeType = process.env['GEN3_AUTH_FILTER_NODE_TYPE'];
-}
 if (process.env['GEN3_PROJECT_ID']) {
   singleton.projectId = process.env['GEN3_PROJECT_ID'];
 }
+
+try {
+  const config_filepath = process.env['ARRANGER_CONFIG_FILEPATH'];
+  const config = JSON.parse(readFileSync(config_filepath).toString());
+  if (config['debug']) {
+    singleton.debug = true;
```

Comment:
how is this used?

Reply:
this was in the config singleton before, but doesn't seem to be used